Description: In the example configuration that is distributed
 as part of this package fix the data.dir to /var/lib/solr/data
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene-solr/solr/example/solr/conf/solrconfig.xml
===================================================================
--- lucene-solr.orig/solr/example/solr/conf/solrconfig.xml	2012-05-28 15:12:31.000000000 +0100
+++ lucene-solr/solr/example/solr/conf/solrconfig.xml	2012-05-28 16:14:42.511797524 +0100
@@ -111,7 +111,7 @@
        replication is in use, this should match the replication
        configuration.
     -->
-  <dataDir>${solr.data.dir:}</dataDir>
+  <dataDir>/var/lib/solr/data</dataDir>
 
 
   <!-- The DirectoryFactory to use for indexes.
Index: lucene-solr/solr/example/solr/conf/scripts.conf
===================================================================
--- lucene-solr.orig/solr/example/solr/conf/scripts.conf	2012-05-23 09:39:26.201829183 +0100
+++ lucene-solr/solr/example/solr/conf/scripts.conf	2012-05-28 16:15:09.332018446 +0100
@@ -15,9 +15,9 @@
 
 user=
 solr_hostname=localhost
-solr_port=8983
+solr_port=8080
 rsyncd_port=18983
-data_dir=
+data_dir=/var/lib/solr/data
 webapp_name=solr
 master_host=
 master_data_dir=
